DTS-STN/canadian-dental-care-plan
# Integrate toater with i18n by using notistack

### Description
Integrate toater with i18n by using [notistack](https://notistack.com/). Replicate PR #330 changes made by @CanadianPhilipBarkow.

### Related Azure Boards Work Items


- [AB#2878](https://dev.azure.com/DTS-STN/1fc40a8f-28cf-47bc-b6e4-1c234bd06177/_workitems/edit/2878)
- [AB#2879](https://dev.azure.com/DTS-STN/1fc40a8f-28cf-47bc-b6e4-1c234bd06177/_workitems/edit/2879)

### Screenshots (if applicable)


![image](https://github.com/DTS-STN/canadian-dental-care-plan/assets/114004123/89e92123-f89c-41fc-9083-cecb3ab7bcb4)

![image](https://github.com/DTS-STN/canadian-dental-care-plan/assets/114004123/37f3d382-d868-4507-8e09-8741c98c7223)

### Checklist


### Test Instructions


Go through change preferred language and phone number flow. When you confirm the changes you should be redirected to `/personal-information` route and a notifiction (toast) message should appears. You should be able to toggle the language and see the message toggle as well.

### Additional Notes
